packages/@divvi/mobile/src/swap/UnfavorableRateBottomSheet.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { RefObject } from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import BottomSheet, { BottomSheetModalRefType } from 'src/components/BottomSheet'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+export default function UnfavorableRateBottomSheet({
+  forwardedRef,
+  onConfirm,
+  onCancel,
+  fromTokenAmount,
+  toTokenAmount,
+}: {
+  forwardedRef: RefObject<BottomSheetModalRefType>
+  onConfirm: () => void
+  onCancel: () => void
+  fromTokenAmount: string // amount with symbol
+  toTokenAmount: string // amount with symbol
+}) {
+  const { t } = useTranslation()
+
+  return (
+    <BottomSheet
+      forwardedRef={forwardedRef}
+      title={t('swapUnfavorableRateBottomSheet.title')}
+      description={t('swapUnfavorableRateBottomSheet.description')}
+      testId="UnfavorableSwapBottomSheet"
+    >
+      <Text style={styles.tokenAmount}>{`${fromTokenAmount} → ${toTokenAmount}`}</Text>
```

Comment:
This should be a translated string too.

Reply:
Ended up doing this differently, using arrow icon instead of the unicode arrow after checking with design.